nuto/mechanics/constraints/Constraints.cpp

```diff
@@ -103,17 +109,6 @@ Eigen::SparseMatrix<double> Constraints::BuildUnitConstraintMatrix(DofType dof,
                 throw Exception(__PRETTY_FUNCTION__,
                                 "There is no dof numbering for a node in equation" + std::to_string(iEquation) + ".");
 
-            if (globalDofNumber >= numIndependentDofs)
-            {
-                // This corresponds to the last block of size numDependentDofs x numDependentDofs that should be an
-                // identity matrix
-                int transformedDof = globalDofNumber - numIndependentDofs;
-                if (transformedDof != iEquation)
-                    throw Exception(__PRETTY_FUNCTION__, "The numbering of the dependent dofs "
-                                                         "is not in accordance with the equation numbering.");
-                continue; // Do not put the value into the matrix.
-            }
-
             if (std::abs(coefficient) > 1.e-18)
                 tripletList.push_back(
                         Eigen::Triplet<double>(numIndependentDofs + iEquation, globalDofNumber, -coefficient));
```

Comment:
This should change to
`Eigen::Triplet<double>(equation.GetDependentDofNumber(), globalDofNumber, -coefficient));`
Seems to be a leftover from the old numbering.